base/static/base/js/index.js

```diff
@@ -1,3 +1,195 @@
-/* Global React components go here. Look at 
-lib_news/static/lib_news/js/NewsFeed.js for 
-an example */
+/* Global React components go here. */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Input = (props) => {
```

Comment:
Ah, so I guess this is part of the global React code because we might want to encode forms as JSON objects outside of the CGIMAIL stuff?

Reply:
Pretty much, however, they wouldn't need to be JSON. The properties could just be passed in manually or come from another data source.